### ISSUE TITLE
access to historical data?

### ISSUE THREAD

**Issue by andreasp888** [tdickman/crypto51#63]
Hello, is it possible to acquire historical data on the cost of a 51% attack for Bitcoin? I would like to use it for academic purposes.

**Comment by klkevelson** [comment]
Yes, but early studies really focused on GHash.IO. Early on, a lot of speculation was published at IETF in RFC's. 
https://trac.ietf.org/trac/irtf/raw-attachment/wiki/blockchain-federation/dirk-blockchain.pdf
